tests/test_standalone_error.py

```
@@ -0,0 +1,29 @@
+                "fn": 1,
+                "args": (1, 2, 3),
+                "kwargs": {"a": 1, "b": 2, "c": 3},
+            }
+        )
+        error_file_content = [
+            'function: 1\n',
+            'args: (1, 2, 3)\n',
+            "kwargs: {'a': 1, 'b': 2, 'c': 3}\n",
+            'ValueError\n'
+        ]
+        with open("error.out", "r") as f:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)